### PR TITLE
Feature/ys update provider table #1540

### PR DESCRIPTION
**Pull request by sivxkxn**
#1540

### CODE REVIEW

**Comment by litvinets** [comment]
1) status color 
image
2) this information should be open on the right side of the table (as it was before), can you please check it out? 
image

File: src/app/shell/admin-tools/data/provider-list/provider-list.component.html [hunk 1]
```diff
@@ -266,4 +266,14 @@
       ></app-provider-info>
     </div>
   </div>
+  <div class="paginator">
+  <app-paginator
+  [totalEntities]="totalEntities"
+  [currentPage]="currentPage"
+  [itemsPerPage]="providersPerPage$ | async"
+  (pageChange)="onPageChange($event)"
+  (itemsPerPageChange)="onItemsPerPageChange($event)">
```
Comment (litvinets) [inline src/app/shell/admin-tools/data/provider-list/provider-list.component.html:275]: aligment

**Review by litvinets — CHANGES_REQUESTED** [review]

File: src/app/shell/admin-tools/data/provider-list/provider-list.component.html (hunk 1 above)
Comment (sivxkxn) [reply src/app/shell/admin-tools/data/provider-list/provider-list.component.html]: fixed

**Comment by litvinets** [comment]
still not fixed 
image

**Review by litvinets — CHANGES_REQUESTED** [review]
image
expected view

**Review by litvinets — APPROVED** [review]

**Pull request merged**